docs/make.jl

```diff
@@ -0,0 +1,25 @@
+push!(LOAD_PATH,"../src/")
+
```

Comment:
This should be removed:
```suggestion
```
Instead, the package should be added in the standard way (as done in the Github action).